experimental/rubyfmt/rubyfmt.cc

```diff
@@ -32,22 +37,31 @@ namespace experimental::rubyfmt {
 
 static int initStatus = RUBYFMT_INIT_STATUS_UNINITIALIZED;
 
+static absl::Mutex mutex;
+
 RubyfmtResult format(string_view str) {
+    // Protect against multithreaded access.
+    absl::MutexLock lock(&mutex);
```

Comment:
This will lock for the duration of the method, does it need that level of synchronization, or is just locking around `rubyfmt_init` enough?

Reply:
Discussed in DM; I'm not sure if it does or not -- I'd have to ask penelope. Initialization is static so it stands to reason that there is shared global state. I was being conservative since there's no downside: formatting is only ever called from one thread in practice anyway.